script/bootstrap

```diff
@@ -67,7 +67,17 @@ install_packages_rpm()
 
 install_packages_brew()
 {
-    echo 'macOS not supported currently' && false
+    # add autotools
+    brew install automake libtool
+
+    # add ARM toolchain
+    brew tap PX4/homebrew-px4
+    brew install gcc-arm-none-eabi
+
+    # check for gcc for posix examples
+    if ! which gcc; then
+        echo 'warning: clang/gcc needed for posix examples - please install xcode to use posix examples'
```

Comment:
They actually just need the CLI tool. You can link to apple: https://developer.apple.com/download/more/